google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb

```diff
@@ -152,21 +168,22 @@ def retrieve_snapshot project, subscription, snapshot_name
       subscription.wont_be :nil?
       subscription.must_be_kind_of Google::Cloud::PubSub::Subscription
       # No messages, should be empty
-      events = subscription.pull
-      events.must_be :empty?
+      received_messages = subscription.pull
+      received_messages.must_be :empty?
       # Publish a new message
       msg = topic.publish "hello"
       msg.wont_be :nil?
       # Check it received the published message
-      events = pull_with_retry subscription
-      events.wont_be :empty?
-      events.count.must_equal 1
-      event = events.first
-      event.wont_be :nil?
-      event.msg.data.must_equal msg.data
-      event.msg.published_at.wont_be :nil?
+      received_messages = pull_with_retry subscription
+      received_messages.wont_be :empty?
+      received_messages.count.must_equal 1
+      received_message = received_messages.first
+      received_message.wont_be :nil?
+      received_message.delivery_attempt.wont_be :nil?
```

Comment:
I wanted to assert that `received_message.delivery_attempt.must_be :>, 0`, but this currently fails with `Expected 0 to be > 0` in all cases. 

The documentation states:

> The first delivery of a given message will have this value as `1`.

Reply:
Can you start with `:">=", 0` for now, add a comment noting the discrepancy, and report a bug to pubsub? (Remember all the GCP issue trackers are linked from https://cloud.google.com/support/docs/issue-trackers)

Reply:
This appears to be working now.

Reply:
So I updated the tests to `received_message.delivery_attempt.must_be :>, 0`